.infrastructure/pvc.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistantVolumeClaim
+metadata:
+  name: pvc-todoapp
+  namespace: todoapp
+spec:
+  storageClassName: standart
```

Comment:
The storage class name 'standart' seems to be a typo. It should likely be 'standard' unless 'standart' is a custom-defined storage class in your Kubernetes cluster.